console/src/components/Services/Data/TableBrowseRows/ViewActions.js

```diff
@@ -57,29 +61,14 @@ const vMakeRequest = () => {
     dispatch({ type: V_REQUEST_PROGRESS, data: true });
 
     const requestBody = {
```

Comment:
Move to v1QueryUtils as `generateSelectQuery = (tableDef, where, limit, offset, orderBy) => { ...`